wondrous-app/components/CreateEntity/CreateEntityModal/index.tsx

```diff
@@ -1284,6 +1286,34 @@ export default function CreateEntityModal(props: ICreateEntityModal) {
     })
   );
 
+  useEffect(() => {
```

Comment:
I think a new request might be an overkill for this, because the same component is also used for editTask modal from the TaskViewModal view which on mount also makes the exact same request. Rewards are passed but not updated due to props.

What I'd suggest is have a hook that changes when existing task reviewers array length changes

why? 
On initial load of the form we set in the initialValues the reviewerIds from the existing task props, but in the meantime a request is made but our initial values are already locked and do not change based on props

As per other we can either populate our BE DTO Model, we already have the fields but they're not populated or make the request when opening the modal from the task card only, because taskviewmodal already has all this data and it can be passed via props

Reply:
We should not overpopulate our DTO model, let's go with the first approach you mentioned

Reply:
made necessary changes @Lamperoyge